src/main/scala/uk/gov/hmrc/agentmtdidentifiers/model/AccessGroupSummary.scala

```diff
@@ -28,19 +28,38 @@ case class AccessGroupSummary(
 
 object AccessGroupSummary {
 
-  def convertCustomGroup(accessGroup: AccessGroup): AccessGroupSummary =
+  def fromAccessGroup(accessGroup: AccessGroup): AccessGroupSummary =
     AccessGroupSummary(
       accessGroup._id.toHexString,
       accessGroup.groupName,
       Some(accessGroup.clients.fold(0)(_.size)),
-      accessGroup.teamMembers.fold(0)(_.size)
+      accessGroup.teamMembers.fold(0)(_.size),
+      None
+    )
+
+  def fromAccessGroup(accessGroup: AccessGroup, clientCount: Int): AccessGroupSummary =
```

Comment:
This should be reusing the TaxGroup instead, so 
`def fromAccessGroup(accessGroup: TaxServiceAccessGroup, clientCount: Int)`

client count is always available for custom groups, but for tax groups we optionally feed in the tax service data when we need it (calculated from the cache + excluded clients once they're added)

Reply:
actually just realised you changed it below, so we just don't need a version here for custom groups